teamA/jeongHaeMan/.gitignore

```
@@ -0,0 +1,194 @@
+# Created by https://www.toptal.com/developers/gitignore/api/dotenv,node,macos,linux
```

`.gitignore` 파일에 현재 프로젝트에 사용되지 않는 요소들이 존재합니다.

불필요한 요소를 제거하고 깃헙에 배포가 되어선 안되는 파일로만 구성해주세요!

```suggestion
.env
node_modules/
db/schema.sql
.vscode 
```